### ISSUE TITLE
training error in cross-validation / forecast

### ISSUE THREAD

**Issue by renefreichel** [facebook/prophet#1865]
I am using prophet in R to make a forecast for one year (360 days) with five years of available data. To evaluate the accuracy of my forecast, I aim to cross-validate it based on training data from four years (1460 days). The "cross-validation" function makes this really convenient:
`df.cv <- cross_validation(m, horizon = 365, units = 'days', initial = 1460)
df.p <- performance_metrics(df.cv)
colMeans(df.p[,3])  # average RMSE = test set error?`

With this piece of code I can easily extract the test set/validation set error (e.g., RMSE) but how do I get to the training error in prophet?

**Comment by bletham** [comment]
Good question - there isn't any utility for computing train error. If you run `fcst <- predict(m)` without any future dataframe passed in then it will make a prediction on the train set, so I think your best bet would be to do that, inner-join it with the training set to add the true `y` column, and then just directly compute RMSE like `sqrt(mean(fcst$y - fcst$yhat) ** 2))`
